cookbook/generate.py

```diff
@@ -0,0 +1,113 @@
+import re
+from pathlib import Path
+from typing import Dict, List
+
+current_dir = Path(__file__).parent.absolute()
+
+input_files = [
+    current_dir / "entrypoints.ts",
+    current_dir / "account.ts",
+    current_dir / "networkProviders.ts",
+    current_dir / "transactions.ts",
+    current_dir / "smartContracts.ts",
+    current_dir / "tokens.ts",
+    current_dir / "accountManagement.ts",
+    current_dir / "delegation.ts",
+    current_dir / "relayed.ts",
+    current_dir / "guarded.ts",
+    current_dir / "addresses.ts",
+    current_dir / "wallets.ts",
+    current_dir / "signingObjects.ts",
+    current_dir / "verifySignatures.ts"
+]
+
+DIRECTIVE_PREFIX = "// md-"
+DIRECTIVE_START = "// md-start"
+DIRECTIVE_IGNORE = "// md-ignore"
+DIRECTIVE_UNINDENT = "// md-unindent"
+DIRECTIVE_AS_COMMENT = "// md-as-comment"
+TO_UNINDENT_SPACE = "    "
+TO_REMOVE = [
```

Comment:
Maybe add a comment why we remove this e.g. _"we don't want this ceremonial piece of code to show up in the rendered cookbook"_.